src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -1246,6 +1256,12 @@ public static string GetDecoratedSymbolName(SymbolReference symbolReference)
                             });
                     }
                 }
+
+                if (processId != null && processId.ToString() != "current") {
```

Comment:
Since you do this comparison a couple times, maybe do a:
```csharp
bool isCurrentProcess = processId != null && processId.ToString() != "current";
```

and use that.

Reply:
Works for me. Done.